### PR TITLE
Release 1.5.0 - Add internalization

### PR DESCRIPTION
**Pull request by victory-sokolov**

### CODE REVIEW

File: package/src/selector.js [hunk 1]
```diff
@@ -1,16 +1,48 @@
...
+    header.appendChild(subHeading);
+
+    mainWrapper.appendChild(obInstitutionSbModalContent);
+    obInstitutionSbModalContent.appendChild(header);
+
+    includeCssFile('https://unpkg.com/flag-icons@6.1.1/css/flag-icons.min.css');
+})();
+
+const obInstitutionSbModalContent = document.getElementById('institution-modal-content');
+const obInstitutionModalHeader = document.getElementsByClassName(
```
Comment (janisskuja) [inline package/src/selector.js:43]: Why not add ID to the header element and get it that way? (Same way you did for `institution-modal-content`)

File: package/src/selector.js [hunk 2]
```diff
@@ -56,7 +93,7 @@ function _institutionSbSetSearchBox(searchBox, config) {
 
     // Set attributes
     search.href = '#';
-    input.setAttribute('placeholder', 'Search...');
+    input.setAttribute('placeholder', `${i18n.search}...`);
```
Comment (janisskuja) [inline package/src/selector.js:96]: A problem for future - what if a translation includes a `.` somewhere in the middle?

I'd suggest using keys like `search` `go_back` `select_your_bank` in `translation.js` and have the values with all the `...` etc..

And if value for key is not available, show the key as is... so it's visible, that there's a missing translation. (alternatively, just make sure `en` has all the translations and fall back to that, if key's not found)

Currently, of course, it works the way it is.

File: package/src/selector.js [hunk 3]
```diff
@@ -214,8 +253,9 @@ function _institutionSbSearchAspsp(config) {
 }
 
 function setOBModalStyles(config) {
+    const obInstitutionSbModalContent = document.getElementById('institution-modal-content');
     const styleConfig = config.styles;
-    const institutionList = Array.from(
+const institutionList = Array.from(
```
Comment (janisskuja) [inline package/src/selector.js:258]: Fix indent

**Review by janisskuja — CHANGES_REQUESTED** [review]
Overall - looks good, just fix the indent, please.

File: package/src/selector.js (hunk 2 above)
Comment (janisskuja) [reply package/src/selector.js:96]: that said, `.` in keys shouldn't be a problem if accessed via `[]` operator... `-` however are a problem

**Review by janisskuja — APPROVED** [review]

**Review by mr-andersons — APPROVED** [review]

**Pull request merged**